core/node/eigenda_proxy/src/errors.rs

```diff
@@ -11,7 +13,17 @@ pub enum MemStoreError {
 pub enum EigenDAError {
     TlsError,
     UriError,
-    ConnectionError,
+    ConnectionError(tonic::transport::Error),
     PutError,
     GetError,
 }
+
+pub(crate) enum RequestProcessorError {
+    EigenDA(EigenDAError),
+}
+
+impl IntoResponse for RequestProcessorError {
+    fn into_response(self) -> Response {
+        unimplemented!("EigenDA request error into response")
+    }
```

Comment:
Shouldn't this be implemented?

Reply:
You are correct, addressed in [680a207](https://github.com/lambdaclass/zksync-era/pull/313/commits/680a207627eef00ac5f619d6ee02e312b116c105)